### ISSUE TITLE
Asking for foreground object mask files of training dataset

### ISSUE THREAD

**Issue by 8uos** [MadryLab/backgrounds_challenge#7]
Hi, thanks for the great work!

I have checked the released training and validation datasets.
It seems like only the foreground mask files of the validation set are released, is it right?
I wonder if I can access the training foreground mask files.
May I ask you to share the files?

Thanks.

**Comment by kaixiao** [comment]
Hi,

Unfortunately, we don't have foreground mask files for the training set available. To obtain masks yourself, you can try running cv2.grabCut on the rectangular region inside each image bounding box.

Let us know if you have any other questions about the process.

**Comment by 8uos** [comment]
Thank you for your answer.
I will try it.
